explorer/testdata/uninitialized/dynaimic/fail_dynamic_unformed_param.carbon

```diff
@@ -0,0 +1,25 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
There shouldn't be much more in "dynamic", but I'd expect a lot more in "static". It may be clearer to distinguish them with subfolders. "control-flow" inside "static" may not worth a subfolder, I'll remove it.